metrics/prometheus/prometheus_test.go

```diff
@@ -1626,3 +1641,34 @@ func TestRecordAdapterGDPRRequestBlocked(t *testing.T) {
 			adapterLabel: string(openrtb_ext.BidderAppnexus),
 		})
 }
+
+func TestStoredResponsesMetric(t *testing.T) {
```

Comment:
Is there a test case you could write where neither metric is incremented?

Reply:
Logic in Prometheus function is the same. I don't know if I can add more TC for this.